test/WebJobs.Script.Tests/Workers/Rpc/TestFunctionRpcService.cs

```diff
@@ -5,7 +5,7 @@
 using System.Collections.Generic;
 using System.Reactive.Linq;
 using Microsoft.Azure.WebJobs.Script.Eventing;
-using Microsoft.Azure.WebJobs.Script.Eventing.Rpc;
+using Microsoft.Azure.WebJobs.Script.Grpc.Eventing;
```

Comment:
Please move tests to a difference project as well in a separate PR.

Reply:
Good call, I can make a different PR altogether after this new grpc work item. I will add an item to the Epic tracking this move.

Reply:
On second thought, we do keep all our tests in the same project, but separate them per the "type" of the test. This may go against that pattern?

Also, this may be one of the only file(s) that could be moved. This project would still need to reference `Webjobs.Script.Grpc` in order for other tests to work properly.

Reply:
Go ahead a open a new issue to refactor the tests. If it does require lot of refactoring to get the tests working, no action needed.